controllers/api/BaseApiController.php

```diff
@@ -148,4 +150,18 @@ private function eagerLoadRelations()
             $this->queryBuilder->with($this->eagerLoad);
         }
     }
+
+    private function applySorting()
+    {
+        $sorts = (array) Request::get('sort', null);
+
+        if (!empty($sorts) && !empty($sorts[0])) {
```

Comment:
Go ahead with `foreach` here without checking if this condition. `(array) null` will return an empty array which `foreach` will "ignore".